src/ansys/fluent/core/systemcoupling.py

```diff
@@ -44,9 +44,10 @@ class SystemCoupling:
     def __init__(self, solver):
         self._solver = solver
         # version check - this requires Fluent 2024 R1 or newer.
-        if FluentVersion.get_latest_installed() < FluentVersion.v241:
+        current_fluent_version = FluentVersion(self._solver.get_fluent_version())
+        if current_fluent_version < FluentVersion.v241:
```

Comment:
Hmm, `get_fluent_version()` should return a `FluentVersion`. The caller can get the `value` attribute which is the string currently returned, I will add a PyFluent issue for this. We don't want to change this at the moment.